packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -176,7 +177,8 @@ Enumerates Groups in your organization with pagination. A subset of Groups can b
 | after     | Specifies the pagination cursor for the next page of Groups                                | Query     | String   | FALSE    |         |
 | filter    | [Filter expression](/docs/reference/api-overview/#filtering) for Groups      | Query     | String   | FALSE    |         |
 | limit     | Specifies the number of Group results in a page                                            | Query     | Number   | FALSE    | 10000   |
-| q         | Searches the `name` property of Groups for matching value                                  | Query     | String   | FALSE    |         |
+| q         | Find groups that matches the `name` property                                               | Query     | String   | FALSE    |         |
+| <ApiLifecycle access="ea" /> search    | Searches for groups with a supported [filtering](/docs/reference/api-overview/#filtering) expression for most properties                        | Query     | String   | FALSE    |         |
```

Comment:
The "Early Access" label/span doesn't visually fit very well here in my opinion:

<img width="883" alt="Screen Shot 2020-09-20 at 6 00 06 PM" src="https://user-images.githubusercontent.com/28817198/93726493-35265b00-fb6b-11ea-96b8-40e723a80865.png">

Chenning or @jakubvul what are your thoughts on the positioning of this? Is there any way to make it fit on the same line (here and in other places on this page where it overflows)?

Reply:
It seems that the tag fits better if followed after the attribute name:
<img width="877" alt="Screen Shot 2020-09-21 at 3 03 29 PM" src="https://user-images.githubusercontent.com/12566503/93826404-01573e00-fc1c-11ea-959b-30241e9a0587.png">

Existing pattern on Apps page:
<img width="880" alt="Screen Shot 2020-09-21 at 3 04 51 PM" src="https://user-images.githubusercontent.com/12566503/93826426-103df080-fc1c-11ea-8871-5ccc32561416.png">

Updated accordingly.